buyer-api/src/app.js

```diff
@@ -22,32 +22,38 @@ import {
 import checkAuthorization from './utils/checkAuthorization';
 
 const app = express();
-app.use(helmet());
-app.use(bodyParser.json());
-app.use(morgan(config.logType || 'combined', {
-  stream: logger.stream,
-  skip: () => config.env === 'test',
-}));
-app.use(cors());
-app.use(boom());
-app.use(bodyParser.urlencoded({
-  extended: true,
-}));
-app.use(cookieParser());
+swagger.initializeMiddleware(schema, ({ swaggerMetadata, swaggerValidator, swaggerUi }) => {
+  app.use(swaggerMetadata());
+  app.use(helmet());
+  app.use(bodyParser.json());
+  app.use(morgan(config.logType || 'combined', {
+    stream: logger.stream,
+    skip: () => config.env === 'test',
+  }));
+  app.use(cors());
+  app.use(boom());
+  app.use(bodyParser.urlencoded({
+    extended: true,
+  }));
+  app.use(cookieParser());
+  app.use(swaggerValidator());
+  // eslint-disable-next-line no-unused-vars
+  app.use((error, req, res, next) => { throw error; });
 
-app.use('/authentication', auth);
-app.use('/health', health);
-app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(schema));
-app.get('/api-docs.json', (_req, res) => res.json(schema));
-app.use('/orders', dataOrders);
-app.use('/notarization-result', notarizationResult);
-app.use('/batch-data-responses', batchDataResponse);
-// This middleware MUST always go after of authentication or fail
-app.use(checkAuthorization);
-app.use('/account', account);
-app.use('/notaries', notaries);
-app.use('/infos', buyerInfos);
+  app.use('/authentication', auth);
+  app.use('/health', health);
+  app.use('/api-docs', swaggerUi());
```

Comment:
```suggestion
  app.get('/api-docs', swaggerUi());
```
It works if we change this